test/models/user_test.rb

```diff
@@ -1,10 +1,45 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
+
+  def setup
+    @user = build :user
+  end
+
   test 'valid factory' do
-    user = build :user
+    assert @user.valid?
+  end
+
+  test 'email should be present' do
```

Comment:
All following test are allready tested by the devise gem itself and can be deleted